gradle/versions.gradle

```diff
@@ -11,7 +11,7 @@ ext.libVersions = [
     jackson: '2.5.1',
     jackson_annotation: '2.5.0',
     cassandra: '2.2.1',
-    cassandra_driver_core: '2.2.0-rc3',
+    cassandra_driver_core: '2.1.5',
```

Comment:
Also, if I recall correctly, the C* 2.1.x driver had a bunch of old, unshaded dependencies for things like Guava that tend to conflict with the rest of the world. Can you include a comment with the diff of gradle dependencies from before/after this change?